src/main/java/wooteco/subway/auth/application/AuthService.java

```diff
@@ -1,7 +1,48 @@
 package wooteco.subway.auth.application;
 
 import org.springframework.stereotype.Service;
+import wooteco.subway.auth.dto.TokenResponse;
+import wooteco.subway.auth.exception.UnauthorizedException;
+import wooteco.subway.auth.infrastructure.JwtTokenProvider;
+import wooteco.subway.member.dao.MemberDao;
+import wooteco.subway.member.domain.Member;
 
 @Service
 public class AuthService {
+
+    private final JwtTokenProvider jwtTokenProvider;
+    private final MemberDao memberDao;
+
+    public AuthService(JwtTokenProvider jwtTokenProvider, MemberDao memberDao) {
+        this.jwtTokenProvider = jwtTokenProvider;
+        this.memberDao = memberDao;
+    }
+
+    public TokenResponse login(String email, String password) {
+        Member member = memberDao.findByEmail(email)
+            .orElseThrow(UnauthorizedException::new);
+
+        validatePassword(password, member.getPassword());
+
+        String jws = jwtTokenProvider.createToken(String.valueOf(member.getId()));
```

Comment:
```suggestion
        String jwt = jwtTokenProvider.createToken(String.valueOf(member.getId()));
```

JWT는 JWS 이외에 다른 형태로 존재할 수 있어 위와 같은 네이밍으로 작성했다고 생각이 듭니다
지금 JWT는 JWS의 구조를 가지고 있는 것은 맞지만, 이는 JWS Compact Serialization의 형태를 띄는 것일 뿐 인증 토큰이라는 의미는 변하지 않습니다
JWS는 암호화를 통해 토큰을 보호하는 JOSE 스펙 중 하나로 추후 JWS 구조가 아닌 JWE 구조로 변경됐을 때 해당 네이밍이 jwe로 바뀌는 것 또한 맞는가? 라는 의문이 있습니다
즉, JWT는 인증 토큰의 형식을 나타내는 의미에 가깝지만 JWS는 인증의 형태와 방식을 나타내는 의미에 가깝기 때문에 jwt라는 네이밍이 더욱 적절하지 않을까 싶은데 현구막의 생각은 어떠신가요 ㅎㅎ

해당 리뷰는 사소한 내용으로 넘어갈 수 있지만 공부할 키워드를 전달하기 위해 길게 남겨보았습니다~!

Reply:
말씀해주신 부분을 보고 곰곰히 생각해보니 자바에서 ArrayList를 선언할 때가 떠오르네요.

```java
ArrayList<Integer> numbers = new ArrayList<>();
List<Integer> numbers = new ArrayList<>();
List<Integer> numbers = new LinkedList<>();
```

게다가 보안 때문에 사용되는 토큰에서 JWS의 이름으로 인증 형태와 방식을 나타낸다는 것은 
**인증 형태와 방식을 노출** 한다는 것과 같은 의미인거 같아요!